requirements.txt

```diff
@@ -0,0 +1 @@
+# generated from manifests external_dependencies
```

Comment:
If you remove this and run pre-commit, is it re generated ?

Reply:
No